DataFormats/GeometryVector/src/classes_def.xml

```diff
@@ -14,15 +14,15 @@
   <class name="VectorTag" ClassVersion="10">
    <version ClassVersion="10" checksum="898711"/>
   </class>
-  <class name="Geom::Phi<float>" ClassVersion="10">
-   <version ClassVersion="10" checksum="2320690959"/>
+  <class name="Geom::Phi<double, Geom::PhiRange::MinusPiToPi>" ClassVersion="10">
+   <version ClassVersion="10" checksum="2898804398"/>
+  </class>
+  <class name="Geom::Phi<float, Geom::PhiRange::MinusPiToPi>" ClassVersion="10">
+   <version ClassVersion="10" checksum="4217567924"/>
   </class>
   <class name="Geom::Theta<float>" ClassVersion="10">
    <version ClassVersion="10" checksum="2498739044"/>
   </class>
-  <class name="Geom::Phi<double>" ClassVersion="10">
```

Comment:
Not clear to me what happens to these ```Geom::Phi<T>```, which are still used in, e.g., ```Basic3DVector.h```

Reply:
@perrotta:
`Geom::Phi<T>` is now equivalent to `Geom::Phi<T, Geom::PhiRange::MinusPiToPi>` since `MinusPiToPi` is the default value for the second parameter.
I tested the dictionary in ROOT, and it recognizes `Geom::Phi<float>` and `Geom::Phi<double>`.
Do you think there will be any problems?

Reply:
Ok, thank you for pointing out the default value for the second parameter!

Reply:
I intended the enhanced `Phi` class to be fully backwardly compatible, so old code would be behave as before. The only exception was the static cast I had to add, as explained above.

Reply:
Is the old Geom::Phi persisted anywhere?
If it is, there should be an effort made to read the old data in a compatible way.

Reply:
The old version of `Geom::Phi` does not appear to be persisted, and even if it was, it can be read by the new code.